cmake/thirdparty/SetupHIP.cmake

```diff
@@ -36,3 +36,26 @@ blt_import_library(NAME          blt::hip_runtime
                    DEPENDS_ON    hip::host
                    TREAT_INCLUDES_AS_SYSTEM ON
                    EXPORTABLE    ${BLT_EXPORT_THIRDPARTY})
+
+if(DEFINED AMDGPU_TARGETS)
```

Comment:
Where is `AMDGPU_TARGETS` defined? 
I don't see it in the host-configs.

Does it come from `find_package(hip )` ?

Would it make sense to add a comment to this file?